salsify_uuid.go

```diff
@@ -0,0 +1,38 @@
+package goavro
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+func salsifyTextualFromNative(buf []byte, d interface{}) ([]byte, error) {
+	switch data := d.(type) {
+	case []uint8:
+		buf = append(buf, "s-"...)
```

Comment:
I think you can represent a single byte with single quotes
```suggestion
		buf = append(buf, 's-')
```

Reply:
Isn't this two bytes?

Reply:
Lol yeah nevermind